### PR TITLE
chore: add qwen2vl-images vqa tool

### PR DESCRIPTION
**Pull request by CamiloInx**
Description
---
* Add qwen2vl-images vqa tool for answering questions on image documents.
* The function accepts list of images to answer questions to. The response is a single string.

Tests
---
Screenshot 2024-10-22 at 2 15 44 PM

### CODE REVIEW

File: vision_agent/tools/tools.py [hunk 1]
```diff
@@ -2162,6 +2194,7 @@ def _plot_counting(
     generate_pose_image,
     closest_mask_distance,
     closest_box_distance,
+    docqa_image,
```
Comment (dillonalaird) [inline vision_agent/tools/tools.py:2197]: remove from FUNCTION_TOOLS for now

**Review by dillonalaird — CHANGES_REQUESTED** [review]
I think we should add QwenVL2 as an image/video QA tool and replace IXC-2.5

File: vision_agent/tools/tools.py (hunk 1 above)
Comment (CamiloInx) [reply vision_agent/tools/tools.py]: Done, sorry, I left it there while testing.

File: vision_agent/tools/tools.py [hunk 2]
```diff
@@ -852,6 +852,38 @@ def ixc25_image_vqa(prompt: str, image: np.ndarray) -> str:
     return cast(str, data["answer"])
 
 
+def docqa_image(prompt: str, image: np.ndarray) -> str:
```
Comment (camiloaz) [inline vision_agent/tools/tools.py:855]: I would change the input from `image: np.ndarray` to `images: list[np.ndarray]` to support multiple page documents in the future, even if the vision-agent does not support it yet.

Comment (CamiloInx) [reply vision_agent/tools/tools.py]: Done

**Review by dillonalaird — APPROVED** [review]
LGTM

**Pull request merged**